packages/expandable/src/ux-expandable.ts

```diff
@@ -0,0 +1,76 @@
+import { customElement, useView, PLATFORM, bindable, TaskQueue, inject } from 'aurelia-framework';
+import { UxComponent, StyleEngine, normalizeBooleanAttribute } from '@aurelia-ux/core';
+import { UxExpandableTheme } from './ux-expandable-theme';
+
+@inject(Element, StyleEngine, TaskQueue)
+@customElement('ux-expandable')
+@useView(PLATFORM.moduleName('./ux-expandable.html'))
+export class UxExpandable implements UxComponent {
+  constructor(public element: HTMLElement, private styleEngine: StyleEngine, private taskQueue: TaskQueue) { }
+
+  static OPEN_CHANGED = 'open-changed';
+
+  header: HTMLElement;
+  content: HTMLElement;
+  contentContainer: HTMLElement;
+
+  @bindable
+  public theme: UxExpandableTheme;
+  public themeChanged(newValue: any) {
+    if (newValue != null && newValue.themeKey == null) {
+      newValue.themeKey = 'expandable';
+    }
+
+    this.styleEngine.applyTheme(newValue, this.element);
+  }
+
+  public openBoolean: boolean = false;
+  @bindable
+  open: boolean | string = false;
+  openChanged() {
+    this.openBoolean = normalizeBooleanAttribute('open', this.open);
+    this.updateContainerHeight();
+    this.element.dispatchEvent(new CustomEvent(UxExpandable.OPEN_CHANGED, { detail: this.openBoolean }));

```

Comment:
if `detail` is simply a boolean, we may later run into a need for a breaking change. Let's do
```ts
detail: { open: this.openBoolean }
```
so if needed, we can add extra properties without any issues. Can also start doing:
```ts
detail { component: this, open: this.openBoolean }
```